indico/cli/event.py

```diff
@@ -85,3 +87,39 @@ def import_(source_file, create_users, force, verbose, yes, category_id):
         sys.exit(1)
     db.session.commit()
     click.secho(event.external_url, fg='green', bold=True)
+
+
+@cli.command('create-series')
+@click.option('--no-title-sequence', is_flag=True,
+              help='Whether to show the series sequence in the event titles (lectures only)')
+@click.option('--no-links', is_flag=True,
+              help='Whether to show links to other events in the series on the event page')
+@click.argument('event_ids', nargs=-1, type=int)
+def create_series(no_title_sequence, no_links, event_ids):
+    """Create a series from a list of events."""
+    events = []
+    for event_id in event_ids:
+        event = Event.get(event_id)
+
+        if event is None:
+            click.secho(f'Event not found (id={event_id}).', fg='red')
+            sys.exit(1)
+        elif event.series is not None:
+            click.secho('Event already assigned to a series:', fg='red')
+            click.echo(format_event(event))
+            sys.exit(1)
+
+        events.append(event)
+
+    click.echo('Selected events:')
+    for event in events:
+        click.echo(format_event(event))
+
+    if click.confirm('Create series?', default=True):
```

Comment:
```suggestion
    click.confirm('Create series?', default=True, abort=True)
```

this will auto-abort in case of no, so you don't need to indent all the code below

Reply:
:+1: